pkgs/servers/web-apps/szurubooru/server.nix

```diff
@@ -50,18 +46,20 @@ python.pkgs.buildPythonApplication {
 
   src = "${src}/server";
 
+  patches = [
+    ./001-server-pillow-heif.patch
```

Comment:
Can't we use `fetchpatch` with https://github.com/rr-/szurubooru/pull/751 (and specify `includes`)?

Reply:
Yes, it is possible. But no, it isn't a good idea to do that. PR's are notorious for being rebased and github is also known to garbage collect these commits since they don't have any references anymore.

See here in https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#patches
> When adding patches in this manner you should be reasonably sure that the used URL is stable. Patches referencing open pull requests will change when the PR is updated and code forges (such as GitHub) usually garbage collect commits that are no longer reachable due to rebases/amends.

Reply:
That's a fair point though it would be easy to retain the commit by pushing it to another branch in your repo.